third_party/angle/include/GLSLANG/ShaderVars.h

```diff
@@ -10,6 +10,8 @@
 #ifndef GLSLANG_SHADERVARS_H_
 #define GLSLANG_SHADERVARS_H_
 
+#include <stdint.h>
```

Comment:
Upstream seems to use `#include <cstdint>`:

https://source.chromium.org/chromium/chromium/src/+/main:third_party/angle/include/GLSLANG/ShaderVars.h;l=15;drc=0510fb4993f06ef7da857bb5c9cef4cc17e0f268